tools/policy_upload_scripts/support_modules/rs_auth.py

```diff
@@ -17,7 +17,7 @@ def rs_oauth(refresh_token):
         shard = shards[shard_idx]
         shard_idx = shard_idx + 1
         try: 
-            uri = "https://us-{}.rightscale.com/api/oauth2".format(shard)
+            uri = "https://moo-93.test.rightscale.com/api/oauth2".format(shard)
```

Comment:
You slippped in a moo-93.test

Reply:
That's strange. I didn't even modify this file.